tests/test.oscrc

```diff
@@ -0,0 +1,11 @@
+[general]
+apiurl = http://Admin:opensuse@localhost:3737
+
+[http://localhost:3737]
+user=Admin
+pass=opensuse
+
+[http://Admin:opensuse@localhost:3737]
```

Comment:
@jberry-suse this was an attempt to force osc to always send basic auth headers - but this failed. osc always sends a reques that ends with 401 to then retry from exception handler with auth.

Do you have an idea how to force direct use of auth? This would cut the size of the cassettes in half :(